website/docs/docs/build/tests.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: "Add tests to your DAG"
-sidebar_title: "Tests"
+sidebar_label: "Tests"
 description: "Read this tutorial to learn how to use tests when building in dbt."
 id: "tests"
+keywords:
+  - test, tests, testing, dag
 ---
-
+# Add tests to your DAG
```

Comment:
sugg removing duplicate -- identical to the page title 

```suggestion
```

Reply:
Oh thanks @nghi-ly ! I added it due to Jason saying something about how an h1 is a level 1 so that's why I added it there. Sorry about not adding more context!

Reply:
no need for sorry, all good!

Reply:
i've been taught that H1 are reserved for page titles. H2 and smaller are used for sections within a page

Reply:
for example, the file https://raw.githubusercontent.com/dbt-labs/docs.getdbt.com/current/website/docs/docs/build/seeds.md

Reply:
gotcha, yes and totally agree with you! i wouldn't usually add an h1 since i also use the title for the page. i added it only to see if the h1 (level 1) helps this page take algolia search precedence over the other faq test pages [per this slck](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1692985181060019?thread_ts=1692979653.610069&cid=C02NCQ9483C)

but perhaps I'm misunderstanding it